### PR TITLE
refactor: 💡 Remove route-action usage for managed-groups

### PR DESCRIPTION
**Pull request by lisbet-alvarez**
✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12619

## Description

Remove route-action usage for managed-groups

## How to Test

1. Run a boundary dev instance.
2. Use existing OIDC and LDAP auth-methods.
3. Test CRUDL operations for managed-groups (create, edit, save, cancel, list, delete)
4. Make sure there are no errors or new warnings in dev tools console.

:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:

- [ ] I have added before and after screenshots for UI changes
- [ ] I have added JSON response output for API changes
- [ ] I have added steps to reproduce and test for bug fixes in the description
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: ui/admin/app/controllers/scopes/scope/auth-methods/index.js [hunk 1]
```diff
@@ -4,8 +4,11 @@
  */
 
 import Controller from '@ember/controller';
+import { action } from '@ember/object';
 
 export default class ScopesScopeAuthMethodsIndexController extends Controller {
+  // =services
+
```
Comment (cameronperera) [inline ui/admin/app/controllers/scopes/scope/auth-methods/index.js:11]: non-blocking: Why was this added?

File: ui/admin/app/routes/scopes/scope/auth-methods/auth-method/managed-groups.js [hunk 2]
```diff
@@ -4,17 +4,12 @@
...
-import { confirm } from 'core/decorators/confirm';
-import { notifySuccess, notifyError } from 'core/decorators/notify';
 
 export default class ScopesScopeAuthMethodsAuthMethodManagedGroupsRoute extends Route {
   // =services
 
   @service store;
-  @service intl;
   @service can;
   @service router;
```
Comment (cameronperera) [inline ui/admin/app/routes/scopes/scope/auth-methods/auth-method/managed-groups.js:14]: It looks like router is no longer being used.

Comment (lisbet-alvarez) [reply ui/admin/app/routes/scopes/scope/auth-methods/auth-method/managed-groups.js:14]: ahh great catch!

File: ui/admin/app/controllers/scopes/scope/auth-methods/index.js (hunk 1 above)
Comment (lisbet-alvarez) [reply ui/admin/app/controllers/scopes/scope/auth-methods/index.js:11]: oh that's not necessary (at least for now), ill remove it.

**Review by calcaide — APPROVED** [review]
Terrific work on `route-actions` 💪

**Review by cameronperera — APPROVED** [review]
